docs/dev/table/functions.md

```diff
@@ -1274,6 +1274,17 @@ ATAN2(numeric1, numeric2)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+COSH(numeric)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns the hyperbolic cosine of <i>numeric</i>.</p>
```

Comment:
This documentation is desperately missing information about returned types. As it is now, user has no idea that for example all `POWER` invocations, even `POWER(2, 5)` yields **SILENTLY** inaccurate results. Same applies to `COSH(2)`, however in the case of `COSH` this is more or less expected. 

I have filled the ticket regarding the already existing functions https://issues.apache.org/jira/browse/FLINK-10438, but could you add such information here for `COSH`?